### PR TITLE
Release candidate 1 for v0.7.5

### PR DESCRIPTION
**Pull request by safl**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,63 @@ Known Issues
...
+    - `{clear,direct,all,save,verbose,ad,idw,idr}*`: changed to `bool`
+    - `action` renamed to `sanact`
+    - `zrms` renamed to `zsa`
+    
+  - `xnvme_pi_generate`
+  
+    - `dbuf` renamed to `data_buf`
+    - `mbuf` renamed to `md_buf`
+
+  - Renamed `enum xnvme_retrieve_opts` to `enum xnvme_retrieve_opts`
```
Comment (karlowich) [inline CHANGELOG.rst:51]: Aren't these identical?

File: CHANGELOG.rst [hunk 2]
```diff
@@ -18,6 +18,63 @@ Known Issues
...
+
+* Library
+
+  - A version-script now controls library-symbols, effectively preventing
+    symbol leakage from linked libraries.
+
+* Infrastructure
+
+  - The ramdisk workflow is now running on Windows and macOS.
```
Comment (karlowich) [inline CHANGELOG.rst:60]: The windows part is still in an open pull request

File: CHANGELOG.rst (hunk 1 above)
Comment (safl) [reply CHANGELOG.rst]: ha, you are right! Fixed.

File: CHANGELOG.rst (hunk 2 above)
Comment (safl) [reply CHANGELOG.rst]: thanks, fixed!

**Comment by karlowich** [comment]
For the changelog, I have a couple of suggestions for meaningful changes to include:
https://github.com/xnvme/xnvme/commit/181ae60d60935847479975838159e00dc27745be
https://github.com/xnvme/xnvme/commit/7f3d1c6a000049834348e9386cfb757b07f3234c
https://github.com/xnvme/xnvme/commit/2ab743f04218d989ce85e7dc2fb59e3bbe560382

**Comment by safl** [comment]
> For the changelog, I have a couple of suggestions for meaningful changes to include: [181ae60](https://github.com/xnvme/xnvme/commit/181ae60d60935847479975838159e00dc27745be) [7f3d1c6](https://github.com/xnvme/xnvme/commit/7f3d1c6a000049834348e9386cfb757b07f3234c) [2ab743f](https://github.com/xnvme/xnvme/commit/2ab743f04218d989ce85e7dc2fb59e3bbe560382)

Good point, added!

**Review by mbrsamsung — CHANGES_REQUESTED** [review]
LGTM

**Comment by safl** [comment]
> LGTM

But you clicked "with comments" and not "approved " ;)

File: CHANGELOG.rst [hunk 3]
```diff
@@ -18,6 +18,81 @@ Known Issues
...
+      version. However, the default behavior now is to disable the SPDK backend
+      instead of failing the build when transient dependencies are missing.
+      This is a much nicer behavior for users who simply grab xNVMe to consume
+      `io_uring` command and NVMe driver ioctl interfaces.
+
+  - Replaced custom shared/static library logic with default Meson handling
+
+    - The default is to build only a single version of the library.
+    - A shared library is the version built by default.
+    - The user can override this and provide '--default-library=both'
```
Comment (karlowich) [inline CHANGELOG.rst:74]: This is not true, it should be:

- The default is to build both the static and shared libraries.
- To build only a single version, the user can use the option: '--default-library' specifying either 'static' or 'shared'

File: CHANGELOG.rst [hunk 4]
```diff
@@ -18,6 +18,81 @@ Known Issues
...
+  - `xnvme_pi_generate`
+  
+    - `dbuf` renamed to `data_buf`
+    - `mbuf` renamed to `md_buf`
+
+  - Renamed `enum xnvme_retreive_opts` to `enum xnvme_retrieve_opts`
+
+* Backends
+
+  - the vfio backend now support iovec payloads
```
Comment (karlowich) [inline CHANGELOG.rst:55]: > the

`The`

**Review by karlowich — CHANGES_REQUESTED** [review]
Couple of nitpicks

File: CHANGELOG.rst (hunk 3 above)
Comment (safl) [reply CHANGELOG.rst:74]: Oh, I thought the was removed:

``default_library=both``

Fixing it. Thanks @karlowich

**Review by mbrsamsung — APPROVED** [review]
lgtm

**Pull request merged**